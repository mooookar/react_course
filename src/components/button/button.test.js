import React from 'react';

import { render, screen } from '@testing-library/react';

// import '@babel/polyfill';

import Button from './button.component';

it('should render a button with right text', () => {
    render(<Button classname="testclass" text="TEST" />);

    expect(screen.getByText('TEST'));
});

it('rendered button snapshot', () => {
    const { container } = render(
        <Button classname="testclass" text="TEST" />
    );
    expect(container.firstChild).toMatchSnapshot()
});
