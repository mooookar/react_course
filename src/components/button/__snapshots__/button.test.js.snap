// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendered button snapshot 1`] = `
<button
  class="testclass"
>
  TEST
</button>
`;
